checklists.md

```diff
@@ -0,0 +1,293 @@
+and can be used to safely speed up execution
+without loss of quality/effectiveness.
+
+> “_under conditions of complexity, not only are checklists a help,
+they are required for success._” <br />
+~ Atul Gawande, The Checklist Manifesto: How to Get Things Right
+
+Atul does a excellent job of explaining
+how several industries/sectors/professions
+develop and use checklists to perform complex high-stakes tasks
+precisely, repeatably and virtually error-free.
+
+There are several good reviews/summaries of the book online.
+One of my favourite summaries is by Nathan Lorenzo (Productivity Game):
+
+[![checklist-manifesto-summary](https://user-images.githubusercontent.com/194400/56947637-b1a73c80-6b25-11e9-9262-9b1d9e851900.png)](https://youtu.be/18ns58FyPmY)
+https://youtu.be/18ns58FyPmY
+
+> Further Reading/Watching on this: <br />
+> "How do we heal medicine?"" | Atul Gawande's TED Talk:
+https://youtu.be/L3QkaS249Bc <br />
+> Extended book review/summary: https://youtu.be/KUv1o-d1m40 <br />
+> NPR "Checklist For Surgery Success":
+https://www.npr.org/templates/story/story.php?storyId=122226184 <br />
+> Praise form several high profile leaders
+[Shane Parrish](https://fs.blog/2014/02/the-checklist-manifesto),
+[Malcolm Gladwell](http://atulgawande.com/book/the-checklist-manifesto),
+[Vinod Khosla](https://twitter.com/vkhosla/status/222469073806700544),
+[Bill Gates](https://www.gatesnotes.com/Books/Checklist-Manifesto), etc.
+
+The book details how checklists were developed independently in
+several industries:
+aviation,
+engineering,
+construction,
+finance,
+medicine,
+etc.
+Without checklists we would not have an aviation industry
+with such a high safety track record.
+see: https://en.wikipedia.org/wiki/Aviation_safety#Evolution
+The same goes for civil engineering where fewer than 1 in 50k
+buildings suffer a failure resulting in death.
+
+
+Ten years later the book remains a "`#1` Best Seller"
+in "Hospital Administration",
+"Health Care Administration"
+and "Family Practice Medicine".
+
+![checklist-manifesto-best-seller](https://user-images.githubusercontent.com/194400/56946012-ec5aa600-6b20-11e9-9630-c225e10dbd25.png)
+https://www.amazon.com/Checklist-Manifesto-How-Things-Right/dp/0312430000
+
+Those categories might not _immediately_ appeal to _you_,
+but consider the fact that the book is "`#954` in Books".
+i.e. it's the `#954` most sold book in the _world_.
+(_for anyone living in a cave for the last 20 years,
+  Amazon sales
+  [charts](https://www.theguardian.com/books/2017/may/22/amazon-charts-books-new-york-times-bestseller-lists)
+  are the ones that matter in 2019!_)
+
+![checklist-manifesto-best-seller-categories](https://user-images.githubusercontent.com/194400/56946615-81aa6a00-6b22-11e9-923e-9eb8ce9171f6.png)
+
+> <sup>1</sup>_**Note**: the book was published in 2009 (as a hardback) but the **paperback** of 2011 is the one that has sold
+the numbers_.
+
+
+# _Why?_
+
+## Why does this _matter_?
```

Comment:
Subtitle rendering issue:
![image](https://user-images.githubusercontent.com/6057298/59186134-08af2100-8b6a-11e9-8a77-6389a4f81cec.png)

Reply:
@SimonLab thanks for taking the time to read the PR and make these comments.
I think I've resolved the Markdown issues.